src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/LoadMetadataCommandHandler.php

```diff
@@ -101,8 +100,6 @@ private function mapTextFields(SaveSamlEntityCommand $command, $metadata)
             'descriptionEn' => ['getDescriptionEn', 'setDescriptionEn'],
             'applicationUrlEn' => ['getApplicationUrl', 'setApplicationUrl'],
             'certificate' => ['getCertificate', 'setCertificate'],
-            'organizationNameEn' => ['getOrganizationNameEn', 'setOrganizationNameEn'],
-            'organizationNameNl' => ['getOrganizationNameNl', 'setOrganizationNameNl'],
```

Comment:
Thanks, I believe the other OrgName occurrences are all Service or Organization VO oriented.